Add further reading section for CAP application deployment automation (#2659)
cap-js/docs: guides/deploy/to-kyma.md

[Learn more about Cloud Native Buildpacks Concepts.](https://buildpacks.io/docs/for-platform-operators/concepts/){ .learn-more}

<div id="aftercloudnative" />

> ## Further reading

Reviewer: The "Deep Dives" section could also be a fitting place for this. It's very little content for a h2 section currently.

Author: good catch, I have added it to deep dives and rephrased little for more context

> For further deployment automation, the [CAP Operator](https://github.com/SAP/cap-operator) provides Kubernetes-native lifecycle management for multi-tenant CAP applications. It introduces Custom Resource Definitions (CRDs) to declaratively describe your application and takes care of tenant provisioning, upgrades, and decommissioning — keeping operations consistent across Kyma and Gardener-managed clusters.

Reviewer 2: The [CAP Operator]((https://sap.github.io/cap-operator/) brings enterprise-grade lifecycle automation to CAP applications — a Kubernetes-native Operator that declaratively deploys and manages application versions, automates tenant operations, and manages domains and networking across Kyma and Gardener-managed clusters.

Author: Hi @Pavan-SAP, I don't want to specify too much detailed external documentation as we can't guarantee to keep the links up-to-date. Do you maybe have a more robust entrypoint for the docs?

Reviewer 2: Sure, I updated it now to: 

The [CAP Operator]((https://sap.github.io/cap-operator/) brings enterprise-grade lifecycle automation to CAP applications — a Kubernetes-native Operator that declaratively deploys and manages application versions, automates tenant operations, and manages domains and networking across Kyma and Gardener-managed clusters.

to also describe briefly what we do and pointed the link to our landing page: https://sap.github.io/cap-operator/

> For further deployment automation, the [CAP Operator](https://github.com/SAP/cap-operator) provides Kubernetes-native lifecycle management for multi-tenant CAP applications. It introduces Custom Resource Definitions (CRDs) to declaratively describe your application and takes care of tenant provisioning, upgrades, and decommissioning — keeping operations consistent across Kyma and Gardener-managed clusters.

Reviewer 3: ```suggestion
For further deployment automation, the [CAP Operator](https://github.com/SAP/cap-operator) provides Kubernetes-native lifecycle management for multitenant CAP applications. It introduces Custom Resource Definitions (CRDs) to declaratively describe your application and manages tenant provisioning, upgrades, and decommissioning. This keeps operations consistent across Kyma and Gardener-managed clusters.
```
